ci-operator/step-registry/configure-registry-storage/set-nutanix-object/configure-registry-storage-set-nutanix-object-commands.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+# This script will set image registry to use nutanix object storage. 
+# It need deploy against nutanix platform.
+# Since nutanix object service still not support terraform, so we will use fixed bucket accordint to OCP version. And will clean up data on nutanix web console periodically.
+
+if [ -f "${SHARED_DIR}/proxy-conf.sh" ] ; then
```

Comment:
I'm not sure what proxy-conf.sh is used for? Maybe we can remove it since this workflow only works on connected network.